api/client/accesslist/accesslist.go

```diff
@@ -113,8 +113,7 @@ func (c *Client) DeleteAccessList(ctx context.Context, name string) error {
 
 // DeleteAllAccessLists removes all access lists.
 func (c *Client) DeleteAllAccessLists(ctx context.Context) error {
-	_, err := c.grpcClient.DeleteAllAccessLists(ctx, &accesslistv1.DeleteAllAccessListsRequest{})
-	return trace.Wrap(err)
+	return trace.NotImplemented("DeleteAllAccesssLists not implemented")
```

Comment:
nit:
I'd change the error message here. This API is our public API, so anyone can use it. Most people are not aware that this is intentional and "X is not implemented" sounds like something that can be implemented in the future. I know that we use the same phrase in all other cases, but I think we should be more explicit about it.